pontoon/actionlog/migrations/0004_auto_20240816_1045.py

```diff
@@ -24,15 +25,21 @@ def migrate_translation_to_actionlog(apps, schema_editor):
 
     actions_to_log = []
 
-    for translation in Translation.objects.filter(date__lt=end_date):
+    translations = Translation.objects.filter(
+        Q(date__lt=end_date)
+        | Q(approved_date__lt=end_date)
+        | Q(unapproved_date__lt=end_date)
+        | Q(rejected_date__lt=end_date)
+        | Q(unrejected_date__lt=end_date)
+    )
+
+    for translation in translations:
         for attr, action_type in traslation_info.items():
             if getattr(translation, attr) is not None:
                 actions_to_log.append(
                     ActionLog(
                         action_type=action_type,
                         created_at=translation.date,
```

Comment:
Also I noticed that since this is actionlog the creation date is not neccessarilly the creation of the translation date but it varies with the  action type:
- approved date
- unapproved date
- rejected date
- unrejected date
- creation date

Should I also change this to match that?